site/docs/components/input/usage.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: Input
+layout: DetailComponent
+sidebar:
+  label: How to use
+data:
+  $ref: ./#/data
+---
+
+### When to use Input
+
+To ask a user a subjective question to which user responses will vary and the length of their answer is likely to be less than a single line. To avoid misleading users, set the width to correlates to the length of the answer you’re expecting. For example, an input for a three-digit value should not be wide enough to accommodate a sentence worth of characters. See [Forms pattern](/salt/patterns/forms) for more information on layout. 
```

Comment:
```suggestion
- To ask a user a subjective question to which user responses will vary and the length of their answer is likely to be less than a single line. 
- To avoid misleading users, set the width to correlates to the length of the answer you’re expecting. For example, an input for a three-digit value should not be wide enough to accommodate a sentence worth of characters. See [Forms pattern](/salt/patterns/forms) for more information on layout. 
```